packages/forms-web-app/src/controllers/eligibility/granted-or-refused-permission.js

```diff
@@ -16,20 +19,38 @@ exports.getGrantedOrRefusedPermission = async (req, res) => {
   });
 };
 
+const forwardPage = (permissionStatus) => {
+  const status = {
+    granted: VIEW.ELIGIBILITY.GRANTED_REFUSED_PERMISSION_OUT,
+    refused: VIEW.ELIGIBILITY.DECISION_DATE,
+    nodecisionreceived: VIEW.ELIGIBILITY.NO_DECISION,
+    default: VIEW.ELIGIBILITY.GRANTED_REFUSED_PERMISSION,
+  };
```

Comment:
Could we use the constant values from https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/feature/as-3292-eligibility-notreceived/packages/forms-web-app/src/constants.js for the object keys rather than hardcoding them in this `status` object?